dist/common/scripts/scylla-housekeeping

```diff
@@ -100,6 +100,7 @@ def version_compare(a, b):
 def create_uuid_file(fl):
     with open(args.uuid_file, 'w') as myfile:
         myfile.write(str(uuid.uuid1()) + "\n")
+        os.fchmod(myfile, 0o644)
```

Comment:
- this is not a valid code...
- why did you use `fchmod`?
use `os.chmod` and send path as a first arg (not file object)

Reply:
![image](https://user-images.githubusercontent.com/5269241/146180892-6752456b-2f81-4e3e-8078-fd9fe59e70b5.png)